opentech/apply/funds/views.py

```diff
@@ -40,15 +49,70 @@ def get_context_data(self, **kwargs):
         )
 
 
-class SubmissionDetailView(DetailView):
+class ProgressSubmissionView(DelegatedViewMixin, UpdateView):
+    model = ApplicationSubmission
+    form_class = ProgressSubmissionForm
+    context_name = 'progress_form'
+
+    def form_valid(self, form):
+        old_phase = form.instance.phase.name
+        response = super().form_valid(form)
+        new_phase = form.instance.phase.name
```

Comment:
I believe this is the same as the lead change

Reply:
Save updates the status which is not changed on the instance until the save method happens, so `old_phase` =/= `new_phase` here.

https://github.com/OpenTechFund/opentech.fund/pull/128/files#diff-9635ffc975a463e0b4204eae3936ff9cR22

Reply:
👍